src/main/java/org/opensrp/util/IdentifierValidatorAlgorithm.java

```diff
@@ -0,0 +1,35 @@
+package org.opensrp.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum IdentifierValidatorAlgorithm {
+	
+	LUHN_CHECK_DIGIT_ALGORITHM("Luhn Check Digit Algorithm");
```

Comment:
any specific reason for this? what advantage it would have in future vs simple Enum..

Reply:
To make it more readable but I have now changed to Simple Enum.